src/views/SideBySideView.js

```diff
@@ -119,6 +118,17 @@ export default class SideBySideView extends VivView {
       getLineColor: viewportOutlineColor,
       getLineWidth: viewportOutlineWidth * 2 ** -thisViewState.zoom
     });
-    return [detailLayer, border];
+    const { PhysicalSizeXUnit, PhysicalSizeX } = loader;
+    const scaleBarLayer =
+      PhysicalSizeXUnit && PhysicalSizeX
+        ? new ScaleBarLayer({
+            id: getVivId(id),
+            loader,
+            PhysicalSizeXUnit,
+            PhysicalSizeX,
+            viewState: thisViewState
+          })
+        : null;
+    return [detailLayer, border, scaleBarLayer];
```

Comment:
It would be clearer to me what's going on if we just pushed layers to an empty array, rather than adding a `null` here. 

```javascript
const layers = [];
// ...
layers.push(detailLayer);
// ...
layers.push(border);
// ... 
if (PhysicalSizeXUnit && PhysicalSizeX) {
  // ...
  layers.push(scaleBarLayer);
}
```